Core/test.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+clear
+npm run test
```

Comment:
Please gitignore, I'd prefer that there is just one obvious way to run tests, which here is `npm run test`. I'm ok to add `clear` there if it doesn't break anything.

Reply:
Removed from repo